velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -88,6 +95,13 @@ class ReferenceQueryRunner {
     return false;
   }
 
+  /// Similar to 'execute' but returns results in RowVector format.
+  virtual std::vector<RowVectorPtr> executeVector(
```

Comment:
ditto

Reply:
SparkQueryRunner does not currently need an implementation of this method.